src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -125,24 +126,24 @@ protected override void ProcessRecord()
             }
         }
 
-        private byte[] ComputeHash(Stream stream)
+        private async Task<byte[]> ComputeHashAsync(Stream stream)
```

Comment:
Why not return just Task and get result with task.GetAwaiter().GetResult()?

Reply:
I mean:
```suggestion
        private Task<byte[]> ComputeHashAsync(Stream stream)
```
No need to use `await` in the method.